genienlp/util.py

```diff
@@ -785,9 +787,8 @@ def load_config_json(args):
             'ned_retrieve_method',
             'database_lookup_method',
             'almond_domains',
-            'ned_features',
-            'ned_features_size',
-            'ned_features_default_val',
+            'almond_type_mapping_path',
+            'max_features_size',
```

Comment:
Do we need compatibility with the old option names in config.json? Otherwise existing trained models will start failing because we won't apply Bootleg correctly.

Reply:
I can add sth that calculates `max_features_size` from `ned_features_deafault_size` and maps the old args to new args.
I imagine we can use the commit hash stored in the config to determine the genienlp version for reevaluating old models?